ios/RNCAsyncStorage.m

```diff
@@ -156,6 +162,52 @@ static dispatch_queue_t RCTGetMethodQueue()
   return error ? RCTMakeError(@"Failed to delete storage directory.", error, nil) : nil;
 }
 
+
+NSString *OldStorageDirectory = @"RNCAsyncLocalStorage_V1";
```

Comment:
Can you add `const` here to prevent accidental overwrite? Also, add a prefix since this lives in the global namespace, and move it to the top along with the other constants.
```objc
NSString *const RCTOldStorageDirectory = @"RNCAsyncLocalStorage_V1";
```